README.md

```diff
@@ -77,7 +77,7 @@ You can find lots of examples at [showcases](https://whs-dev.surge.sh/examples/)
     const app = new WHS.App([
       new WHS.ElementModule(), // attach to DOM
       new WHS.SceneModule(), // creates THREE.Scene instance
-      new WHS.CameraModule(), // creates PerspectiveCamera instance
+      new WHS.DefineModule('camera', new WHS.PerspectiveCamera()), // creates PerspectiveCamera instance
```

Comment:
we have to be careful with updating readme, (well doc too if we publish early) as this module is not available with beta releases yet.